authproxy/Dockerfile

```diff
@@ -0,0 +1,13 @@
+FROM nginx:stable-alpine
```

Comment:
This shouldn't be checked into doccano. Remove and add as a reference to the pull request description.

Reply:
I have removed the authproxy directory and added relevant files in the description of the PR.